src/style.css

```
@@ -0,0 +1,30 @@
+body {
+  margin: 0;
+  min-height: 100vh;
+}
+
+.stars {
+  margin: auto;
+  display: flex;
+  justify-content: center;
+}
```

we are going to re-use this block, do it shouldnt be any unnecessary styles

```suggestion
.stars {
  display: flex;
}
```